### ISSUE TITLE
Fix partial import of CRIS data for today and going forward

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#12053]
The prefect agent reported today that there is a problem serializing a number being represented using the Decimal class. This has resulted in a partial import of crashes, where we have some new crashes, but do not yet have the units, persons and primary persons associated with them. 

This needs to be remediated by rebuilding the encrypted zip file from the CSVs we have in S3 and run through the import process to debug the problem in the flow, and then finally, one last time to actually populate the entire dataset into our database.

**Comment by frankhereford** [comment]
Xavier reached out this afternoon for an update, and I provided him this which is also a summary of the progress and situation:

> Thanks for writing. It’s actually quite a wrinkle that CRIS has thrown us, but I’ve been working it all day, and I anticipate have the import up and running today so that we can get today’s export in and be on schedule for tomorrow. For a little behind the scenes, CRIS has split their exports into multiple schemas or forms of the export. These have to be handled differently, but I’ve been working on making a map/reduce safe ETL that can handle the multiple ways they are exporting data. It’s looking good!

**Comment by frankhereford** [comment]
Something just occurred to me here that should be recorded. This makes the time discrepancy clear and emphasizes the reason for the backfill in #12064 even more needed.

* The bug to this work today was the error `TypeError: Object of type Decimal is not JSON serializable` which was causing the ETL to fail. This is actually in a routine that we pull in from the oooold ETL code in `atd-vz-data` and is related to the "conflict resolution system / change log" we were talking about earlier in planning.
* In investigating what was going on, I discovered that CRIS has gone split-schema on us, and we were not correctly handling the pre-vs-post April 1st schema change in our ETL. This is what I have worked on almost all day, making it map/reduce safe to handle multiple schemas in a serialized way which is safe for the DB.
* While doing this, I'll fix the TypeError bug too. Two birds, one stone.
